auth0/structure_auth0_connection_test.go

```diff
@@ -0,0 +1,15 @@
+package auth0
+
+import "testing"
+
+func TestFlattenConnectionOptions(t *testing.T) {
```

Comment:
I always wondered why we didn't unit test the flatten function because there is quite a bit of logic and complexity wrapped up in them. However, this test is just asserting the simple case of a `nil` options struct which should return `nil`.